rest_framework_nested/viewsets.py

```diff
@@ -39,28 +49,28 @@ def _get_parent_lookup_kwargs(self) -> dict:
 
         return parent_lookup_kwargs
 
-    def get_queryset(self):
+    def get_queryset(self) -> QuerySet[T_Model]:
         """
         Filter the `QuerySet` based on its parents as defined in the
         `serializer_class.parent_lookup_kwargs` or `viewset.parent_lookup_kwargs`
         """
-        queryset = super().get_queryset()
+        queryset = super().get_queryset()  # type: ignore[misc]
```

Comment:
Is this not acceptable?

```suggestion
        queryset: QuerySet = super().get_queryset()  # type: ignore[misc]
```

Reply:
The mypy error is "error: "get_queryset" undefined in superclass", because this is a mixin class and mypy doesn't know what the bases are

Elsewhere I have solved this with a cast

```suggestion
        queryset = cast(ModelViewSet, super()).get_queryset()
```

Reply:
I see. In this case, I vote for `# type: ignore[misc]` everywhere the `cast()` appeared.